Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -262,6 +262,50 @@ public class ChatChannelController: DataController, DelegateCallable, DataStoreP
         }
     }
 
+    /// Updates  channel information with provided data, and removes unneeded properties.
+    ///
+    /// - Parameters:
+    ///   - team: New team.
+    ///   - members: New members.
+    ///   - invites: New invites.
+    ///   - extraData: New `ExtraData`.
+    ///   - unsetProperties: Properties from the channel that are going to be cleared/unset.
+    ///   - completion: The completion. Will be called on a **callbackQueue** when the network request is finished.
+    ///                 If request fails, the completion will be called with an error.
+    ///
+    public func partialChannelUpdate(
+        name: String? = nil,
+        imageURL: URL? = nil,
+        team: String? = nil,
+        members: Set<UserId> = [],
+        invites: Set<UserId> = [],
+        extraData: [String: RawJSON] = [:],
```

Comment:
Nit: This is probably to keep it consistent with the regular `updateChannel()`, but ideally, all these properties should be a struct. Something like `struct ChannelUpdateOptions { var name; var iamgeURL, etc.. }`, otherwise, the function arguments will keep increasing in the future. This struct could then be reused for the `channelUpdate()` as well.